sdk-base/src/main/java/com/mapbox/maps/plugin/annotation/GeoJsonOptions.kt

```diff
@@ -0,0 +1,68 @@
+package com.mapbox.maps.plugin.annotation
+
+/**
+ * Configure class for composing GeoJsonSource objects that included in Annotation
+ * @see <a href="https://docs.mapbox.com/mapbox-gl-js/style-spec/sources/#geojson">The online documentation</a>
+ */
+data class GeoJsonOptions(
+  /**
+   * The minimum zoom level for the layer. At zoom levels less than the minzoom, the layer will be hidden.
+   * Range: [0, 24]
+   */
+  val minZoom: Double? = null,
+
+  /**
+   * The maximum zoom level for the layer. At zoom levels equal to or greater than the maxzoom, the layer will be hidden.
+   * Range: [0, 24]
+   */
+  val maxZoom: Double? = null,
```

Comment:
the configurations in this class should only impact the source configuration. We could broaden this but if so, we need to qualify what maxZoom/minZoom you are changing layer or source.